lib/SILOptimizer/Mandatory/MandatoryInlining.cpp

```diff
@@ -175,11 +187,22 @@ static void fixupReferenceCounts(
     // TODO: Do we need to lifetime extend here?
     case ParameterConvention::Direct_Unowned: {
       v = SILBuilderWithScope(pai).emitCopyValueOperation(loc, v);
-
       visitedBlocks.clear();
+
       // If we need to insert compensating destroys, do so.
+      //
+      // NOTE: We use pai here since in non-ossa code emitCopyValueOperation
```

Comment:
NOTE: When I add the new API, I am going to eliminate these comments. Since I am using this for cherry-picking, I wanted to be more explicit.